codice/ddf
# DDF-4080 Add degrees minutes seconds support for bounding box and point-radius searches to Intrigue's search builder

#### What does this PR do?
Adds a "Lat/Lon (DMS)" tab to the bounding box and point-radius editors in Intrigue's search builder. This allows users to specify bounding boxes and point-radius searches using degrees minutes second coordinates.

#### Who is reviewing it? 
@SmithJosh 
@leo-sakh 
@bellcc

#### Select relevant component teams: 
@codice/ui 

#### Ask 2 committers to review/merge the PR and tag them here.
@andrewkfiedler
@djblue

#### How should this be tested?
Build DDF and open Intrigue. Perform both bounding box and point-radius geospatial searches by entering degrees minutes seconds coordinates manually. Verify that the bounds are drawn on both the Cesium and OpenLayers maps correctly. Run the search and verify that only results within the bounds are returned. On both the Cesium and OpenLayers maps, draw bounding box and point-radius searches and verify that the degrees minutes seconds values are correct. Switch between the different coordinate formats (DD, DMS, USNG/MGRS, and UTM) and verify that the values are converted correctly.

#### What are the relevant tickets?
[DDF-4080](https://codice.atlassian.net/browse/DDF-4080)

#### Screenshots
![dmsboundingbox](https://user-images.githubusercontent.com/4495447/44478686-42229a00-a5f3-11e8-94bc-217a092f0725.jpg)
![dmspointradius](https://user-images.githubusercontent.com/4495447/44478688-42229a00-a5f3-11e8-9552-741279ba5852.jpg)

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
